packages/kbn-aiops-utils/src/lib/string_reducer.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Reducer, ReducerAction, ReducerState } from 'react';
+
```

Comment:
Small suggestion to add a comment here or in the string stream example documenting that users can pass in arrays to do bulk updates as an optimization.

Reply:
Improved documentation for `stringReducer()` here: 
e80dffc.